platform/viewer/src/utils/cornerstoneWADOImageLoader.js

```diff
@@ -0,0 +1,57 @@
+import OHIF from '@ohif/core';
+import cornerstone from 'cornerstone-core';
+import dicomParser from 'dicom-parser';
+
+export function getCornerstoneWADOImageLoader() {
+  return import(
+    /* webpackChunkName: "CornerstoneWADOImageLoader" */ 'cornerstone-wado-image-loader'
+  );
+}
```

Comment:
Hmm.. Maybe `getCornerstoneWADOImageLoader` should call `initCornerstoneWADOImageLoader` if it hasn't been initialized yet? Then we don't need to export or worry about when/what calls `initCornerstoneWADOImageLoader`?